dbcon/mysql/ha_mcs_sysvars.h

```diff
@@ -34,12 +34,19 @@ enum mcs_compression_type_t {
 };
 
 // use_import_for_batchinsert
-enum mcs_use_import_for_batchinsert_t {
+enum class mcs_use_import_for_batchinsert_t {
```

Comment:
I would rename this one also but it goes outside of the scope.

Reply:
Also updated this one.